examples/jsm/loaders/GLTFLoader.js

```diff
@@ -3197,9 +3197,7 @@ class GLTFParser {
 			if ( ! pointsMaterial ) {
 
 				pointsMaterial = new PointsMaterial();
-				Material.prototype.copy.call( pointsMaterial, material );
-				pointsMaterial.color.copy( material.color );
-				pointsMaterial.map = material.map;
+				pointsMaterial.copy( material );
```

Comment:
I don't think this line of code is right. `Material.prototype.copy.call( pointsMaterial, material );` only executes the `copy()` method of the `Material` class whereas `pointsMaterial.copy( material );` executes `PointsMaterial.copy()`. However, this will lead to `undefined` material properties if the given material is not of type `PointsMaterial`.

Reply:
> Remove prototype accesses

Like I said before, `prototype` access is not evil. At some points, it does make sense and a refactoring is not really required.

Reply:
OK, reverted this change:
https://github.com/mrdoob/three.js/pull/24190